substrate/frame/contracts/uapi/src/host/wasm32.rs

```diff
@@ -650,6 +709,23 @@ impl HostFn for HostFnImpl {
 		ret_code.into()
 	}
 
+	#[inline(always)]
```

Comment:
Is there any rule for when we should annotate a function as inline here?

Reply:
my approach is to only add it if this make a significant diff in the benchmarks